docs/content/contributing/index.md

```diff
@@ -4,6 +4,14 @@ Thank you for your interest in contributing to the AWS PDK. Whether it's a bug r
 
 Please read through this document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your bug report or contribution.
 
+## Getting Started
+
+Fork the repository and clone it to your local machine.
+```
+$ pnpm i && pnpm build
+$ pnpm watch
```

Comment:
I don't think we need `pnpm watch` here, simpler to just include the install and build commands :)